command/operator_debug.go

```diff
@@ -1064,6 +1064,10 @@ func (e *external) addr(defaultAddr string) string {
 		if strings.HasPrefix(e.addrVal, "http:") {
 			return e.addrVal
 		}
+		if strings.HasPrefix(e.addrVal, "https:") {
```

Comment:
I was going to leave a comment about this being an awfully weird addressing scheme to support, but then I looked it up the details of the caller and apparently Consul supports TLS over UDS. So... ok then! 😀